### DISCUSSION TITLE
Payments integration

### DISCUSSION THREAD

**Discussion by aleksadjukic** [adonisjs/core#3023] [General]
I recently started working with AdonisJS and I have to say I really like it so far. It is very similar to Laravel, which I think is awesome.
However, I would suggest payments integration into the framework. Something like Cashier for Laravel, which would provide a built-in module for handling payments using Stripe, PayPal etc. 
I believe this would give the framework an edge over the other Node frameworks.

**Comment by McSneaky** [comment]
Seems like Mezie has already started with community package for Stripe
https://github.com/mezielabs/adonis-stripe

You can investigate how it works and make PayPal one based on that or open RFC issue in there and then perhaps can extend it to something like `adonis-payments`
